### ISSUE TITLE
Reference a favicon to prevent backend 404s

### ISSUE THREAD

**Issue by dan2468** [HangfireIO/Hangfire#1885]
Could you please include something like a
``` html
<link rel="shortcut icon" href="data:image/x-icon;," type="image/x-icon">
```
line in the `<head>` of `\src\Hangfire.Core\Dashboard\Pages\LayoutPage.cshtml` (I think that this will regenerate `LayoutPage.cshtml.cs`), to prevent the default browser behaviour of trying to load `/favicon.ico` and trigger a “404 (Not Found)” error on the backend service immediately after opening the `/hangfire` dashboard page ?

**Comment by odinserj** [comment]
Sure, will be fixed in 1.7.24!

**Comment by scidec** [comment]
Still not working in 1.7.33
